.gitignore

```diff
@@ -1,5 +1,7 @@
 /node_modules/
 /build/
+/browserify/
```

Comment:
This line is unnecessary.

With the change of GPII-2630, the directory for holding browserified js files has been moved to `build/tests/browserify`. See [scripts/browserifyTestDependency.js](https://github.com/GPII/universal/blob/master/scripts/browserifyTestDependency.js#L37). The residue of the old `/browserify` can be manually removed from the local development env.